lib/SiftResponse.php

```diff
@@ -23,6 +23,7 @@ public function __construct($result, $httpStatusCode, $request) {
         $this->httpStatusCode = $httpStatusCode;
         $this->request = $request;
         $this->rawResponse = $result;
+        $this->apiErrorMessage = $result->error
```

Comment:
I think you meant `$request->error`, but can you move `error` out of the request class? That's what I meant by putting the error message in a parameter to `__construct`.